### PR TITLE
Example instructions on configuring Apache for iOS Universal Link support

### PR DESCRIPTION
**Pull request by SamuAlfageme**
## Do Not Merge Yet

This is not final. We should check with someone with Apache skillzs 'bout the rules/config I used here for production environments (@butonic maybe?) . Also a review from @owncloud/doc would be nice. 

I moved old sections 8 and 9 to 9 and 10 respectively. All the new content is in `publishing_ios_app_8`: https://github.com/owncloud/branded_clients/blob/cd99beb054fbd8bef9e30948cf75e046cfdc9cd6/branded_ios_app/publishing_ios_app_8.rst

Ref. 
- https://github.com/owncloud/enterprise/issues/2173 
- https://github.com/owncloud/ios/issues/972

### CODE REVIEW

**Comment by SamuAlfageme** [comment]
- [ ] Add note on how some firewall/proxies can be used instead of ownCloud's webserver to get the required content served.

File: branded_ios_app/publishing_ios_app_8.rst [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+ Adding support for universal links is easy. There are three steps you need to take:
+
+    - Create an apple-app-site-association file that contains JSON data about the URLs that your app can handle.
+    - Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the .well-known subdirectory.
+    - Prepare your app to handle universal links. 
+
+The `apple-app-site-association` will be generated by ownbrander and needs to be served statically on your site through HTTPS. You can safely place it on the root folder of your ownCloud installation (e.g. `/var/www/owncloud`). To achieve the second requirement, some changes will also need to be done on your Apache configuration.
+
+If you configured your installation with the official `Admin Manual
+<https://doc.owncloud.org/server/10.0/admin_manual/installation/source_installation.html#apache-configuration-label>`_, your `owncloud.conf` file should include the following:
```
Comment (mmattel) [inline branded_ios_app/publishing_ios_app_8.rst]: Must be a xref not a web link.
For details see best practices in readme.md

Comment (settermjd) [reply branded_ios_app/publishing_ios_app_8.rst]: This needs to be converted to AsciiDoc and moved to the `master-antora` branch. I'll get it sorted.

**Comment by settermjd** [comment]
Closing, as it's out of date.

**Pull request closed without merging**